geo/map_processing/map_processor.py

```diff
@@ -21,14 +21,16 @@
 
 FLAGS = flags.FLAGS
 flags.DEFINE_string("place", None, "map area - Manhattan or Pittsburgh.")
-flags.DEFINE_string("path", None, "path to save the map file.")
+flags.DEFINE_integer("level", None, "min level of map")
 
 # Required flags.
 flags.mark_flag_as_required("place")
 
 
 def main(argv):
     del argv  # Unused.
+    pittsburgh_map = Map(FLAGS.place, level)
```

Comment:
Shouldn't be specific to pittburgh, so should be just 'map = '. That said, there isn't anything done with the map, so you could just use Map(FLAGS.place, level) without assignment. (Or, better, do a thing or two with the map after it is created.)

Reply:
done